### PR TITLE
Edit JIAOZI space

### PR DESCRIPTION
**Pull request by nightg0at**

### CODE REVIEW

**Comment by e-asia** [comment]
@nightg0at You will want to add logo.png and space.png files.
https://docs.snapshot.page/guides/create-a-space#4-add-a-logo-and-space-images

**Comment by nightg0at** [comment]
Hi @e-asia, the files you mention are already in your repo (see #39). This is an edit not a new space.

**Review by e-asia — APPROVED** [review]

**Comment by e-asia** [comment]
Hi @nightg0at. Yes you are right, sorry my bad, reviewed and approved.

File: spaces/jiaozi/index.json [hunk 1]
```diff
@@ -5,7 +5,14 @@
...
+  ],
   "min": 0,
-  "invalid": []
-}
+  "invalid": [
+    "QmUF7aBnmfuLwC29XtHT6zxkeREm8p8FQb4BVSgZG15s66",
+    "QmYnvLtMsBmSWTNp5iUKEZDXbhTCexM8BEzwW24X3kpJdR",
+    "QmP5C6rQcurbk6yvX9CZhxArQktHBykhYnHUotoufHB8pn"
+  ]
+}
```
Comment (e-asia) [inline spaces/jiaozi/index.json:18]: We have updated the format in `index.json`. 
Please add 2 new parameters `chainId` and `decimals`.
You can refer to the documentation that has been completed. https://docs.snapshot.page/guides/create-a-space#3-change-your-space-metadata

**Review by e-asia — CHANGES_REQUESTED** [review]

**Comment by e-asia** [comment]
Closing this PR for now, feel free to open it again once you've done the changes.

**Pull request closed without merging**